### ISSUE TITLE
Unstyled Option buttons

### ISSUE THREAD

**Issue by Mottie** [RaitaroH/Stylus-DeepDark#2]
In Chrome, the option setting buttons are unstyled

![](https://user-images.githubusercontent.com/136959/33141631-600c269a-cf79-11e7-861c-b73f25c97c78.gif)

**Comment by RaitaroH** [comment]
Hmmm
Let me take a look.

**Comment by RaitaroH** [comment]
Ok, so:
![screenshot from 2017-11-22 21 05 15](https://user-images.githubusercontent.com/19552720/33145399-e0b9d4c8-cfc8-11e7-9522-1225ffeb4fb9.png)

I tried this:
```
:-webkit-any(button,input[type="button"],input[type="submit"]), select, input[type="checkbox"], input[type="radio"]
	{
		background-color: var(--second-background) !important;
		color: var(--main-text) !important;
		border: 1px solid var(--hover-background) !important;
		box-shadow: 0 1px 0.5px rgba(0,0,0,0.13) !important;
	}
``` 
It doesn't work. 
I tried just `button` doesn't work. 
I tried `button[data=""]` doesn't work.

**Comment by Mottie** [comment]
LOL no worries, get some rest. I can look at it later and submit a PR.

**Comment by RaitaroH** [comment]
Ok. Thanks.

**Comment by RaitaroH** [comment]
I knew about this tho. I just thought at the time stylus cannot style those because of some bug or API crap.

**Comment by RaitaroH** [comment]
'Night then!

**Comment by FranklinYu** [comment]
I think similar bug occurs again, except that this time the background is white, not the button.

**Comment by the-j0k3r** [comment]
Stylus 1.5.7 changes a few elements styling, Ive already got a fix for this and also other fixes for this popup in chrome + other tab navigation fixes.

Soon as I can test stylus 1.5.7 in Firefox (its not been updated yet so manual loading is necessary) Ill push fixes through. It likely wont be today, so be patient.

Also next time if the bug is not for same issue feel free to open a new issue. This time I'll reopen this one though.

Thanks.

**Comment by the-j0k3r** [comment]
@FranklinYu I pushed some fixes in 2.2.2 still waiting fro Firefox to update Stylus to 1.5.7 to see if anything needs fixing there.
